src/ILLink.Tasks/ILLink.Tasks.csproj

```diff
@@ -101,7 +100,7 @@
     <Compile Include="LinkTask.cs" />
     <Compile Include="CheckEmbeddedRootDescriptor.cs" />
     <Compile Include="CompareSizes.cs" />
-    <Compile Include="ComputeCrossgenedAssemblies.cs" />
+    <None Include="ComputeCrossgenedAssemblies.cs" />
```

Comment:
Can we delete ComputeCrossgenedAssemblies.cs and remove the <None> entry from here?

Reply:
I'd like to keep the code around for now, in case we want to do anything special with crossgen'd assemblies. I included it in None as a reminder to myself to either compile it in or remove it entirely when we have settled on concrete linker default.

Reply:
I think we can always get it back from the git history if we ever needed to, but it's confusing keeping it around and not compile it

Reply:
I agree the file can be always found in the history log

Reply:
Removed along with some other files that are no longer needed.